src/functions-reference/higher-order_functions.Rmd

```diff
@@ -126,14 +144,23 @@ solver directly to the system function.
 
 #### Algorithmic details
 
-The algebraic solver uses the Powell hybrid method [@Powell:1970],
+Stan offers two algebraic solvers.
+The first one, `algebra_solver`, uses the Powell hybritd method [@Powell:1970],
```

Comment:
I find calling them "first" and "second" odd - it refers to historical order, yeah, but that really does not matter to the reader. Maybe instead "One solver is the Powell"... "The other solver is..."